src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -44,8 +47,14 @@ public AddCommand parse(String args) throws ParseException {
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+        Priority priority;
+        if (argMultimap.getValue(PREFIX_PRIORITY).isEmpty()) {
```

Comment:
@nigellqx the argumentMultimap::getValue method returns an optional. I think it would be a better idea to make use of the methods provided by the optional class in Java. Maybe you can try something like this.
```
Priority priority = ParserUtil.parsePriority(
    argMultimap.getValue(PREFIX_PRIORITY).orElse("low")
);
```

Reply:
Thank you for your comment. I agree with you that using Optional class is better in this scenario. I will make the necessary changes.